ext/mbstring/mbstring.c

```diff
@@ -300,15 +308,15 @@ static zend_result php_mb_parse_encoding_list(const char *value, size_t value_le
 	} else {
 		bool included_auto;
 		size_t n, size;
-		char *p1, *endp, *tmpstr;
+		const char *p1, *endp, *tmpstr;
 		const mbfl_encoding **entry, **list;
 
 		/* copy the value string for work */
```

Comment:
Looks like this comment is now redundant, not so?

Reply:
Good catch!